### ISSUE TITLE
Add click-ability to README icon list

### ISSUE THREAD

**Issue by chrisgereina** [tailwindlabs/heroicons#66]
First off, thanks for making this free and great job.

When I landed on the README I assumed (naively) that I would be able to click on the icon I desired and would then be directed to a snippet of code that I could copy and paste into my project.

I know that it would likely involve quite a bit of work to do this, but that would be killer functionality that would pair well with the stuff you guys have done on Tailwind UI (side note, super amazing job on that by the way. I'm already using it for a client and it is top-notch)

Thanks again

Edit: realized on my second pass that there's a website listed on the README where exactly this is possible: https://heroicons.dev/ -- still might be worth making that more prominent or linking the icon list image directly to that site, so going to leave this open in the interest of UX :laughing:

**Comment by adamwathan** [comment]
Hey thank you! Check out heroicons.dev I bet that’ll solve your issue 👍🏻
On Sat, Jun 20, 2020 at 9:00 PM GitHub <notifications@github.com> wrote:

“First off, thanks for making this free and great job.

When I landed on the README I assumed (naively) that I would be able to click on the icon I desired and would then be directed to a snippet of code that I could copy and paste into my project.

I know that it would likely involve quite a bit of work to do this, but that would be killer functionality that would pair well with the stuff you guys have done on Tailwind UI (side note, super amazing job on that by the way. I'm already using it for a client and it is top-notch)

Thanks again”
